libs/parallelism/algorithms/include/hpx/parallel/algorithms/remove_copy.hpp

```diff
@@ -35,10 +35,10 @@ namespace hpx { namespace parallel { inline namespace v1 {
         /// \cond NOINTERNAL
 
         // sequential remove_copy
-        template <typename InIter, typename OutIter, typename T, typename Proj>
+        template <typename InIter, typename Sent, typename OutIter, typename T,
+            typename Proj>
         inline util::in_out_result<InIter, OutIter> sequential_remove_copy(
```

Comment:
This algorithm can be marked as `constexpr`.

Reply:
Do we always declare our sequential algorithms as `constexpr` (for optimization reasons maybe)?

Reply:
Yes, if possible.

Reply:
already have... sorry for not putting this as resolved ;)